### ISSUE TITLE
How can I convert tflite model?

### ISSUE THREAD

**Issue by zmqp111** [experiencor/keras-yolo2#411]
hi 

I am going to convert Keras model into tflite model.

But, I have some problems.

1. cannot use custom function.
    - In some cases, the CustomObjectScope function is applied, but not applied, as a result of experimenting with changing the version of keras and tensorflow.(Keras : 2.1.6, TF : 1.13.1 ===> can)
   - Cannot use custom_loss() in frontend.py

2. What is input arrays and output arrays?

3. How can I convert Keras model to tflite model?

thans

**Comment by rodrigo2019** [comment]
1º save your model
2º create a second yolo model
3º load the weights from the saved model

in this way your model won't be compiled, so you don't need to set the custom loss
